app/assets/src/components/views/discovery/DiscoverySidebar.jsx

```diff
@@ -171,21 +177,23 @@ export default class DiscoverySidebar extends React.Component {
     return rows.map((entry, i) => {
       const { count, text, value } = entry;
       const percent = Math.round(100 * count / total, 0);
-      return [
-        <dt className={cs.barLabel} key={`${value}_label_${i}`}>
-          <a onClick={() => this.handleFilterClick(value)}>
-            {value === "not_set" ? <i>{text}</i> : text}
-          </a>
-        </dt>,
-        <dd key={`${value}_value_${i}`}>
-          <span
-            className={cs.bar}
-            // TODO(gdingle): make width depend on container
-            style={{ width: percent * 1.3 + "px" }}
-          />
-          <span className={cs.count}>{count}</span>
-        </dd>
-      ];
+      return (
+        <div className={cs.barChartRow} key={`${value}_row_${i}`}>
+          <dt className={cs.barLabel} key={`${value}_label_${i}`}>
+            <a onClick={() => this.handleFilterClick(value)}>
+              {value === "not_set" ? <i>{text}</i> : text}
+            </a>
+          </dt>
+          <dd key={`${value}_value_${i}`}>
+            <span
+              className={cs.bar}
+              // TODO(gdingle): make width depend on container
```

Comment:
you can probably remove this comment now

Reply:
👍